src/foam/nanos/jetty/HttpServer.js

```diff
@@ -59,10 +59,18 @@ foam.CLASS({
       name: 'start',
       javaCode: `
       try {
-        System.out.println("Starting jetty http server.");
+        System.out.println("Starting Jetty http server.");
         org.eclipse.jetty.server.Server server =
           new org.eclipse.jetty.server.Server(getPort());
 
+        for ( org.eclipse.jetty.server.Connector conn : server.getConnectors() ) {
```

Comment:
Add comment to explain what this does/is-for.

Reply:
Added a comment.